CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.5.3 (TBD)
+
+- Feature: Client-side binaries for the amd64 architecture are now available for linux
```

Comment:
Typo in the changelog, that should be arm64

Reply:
lol thank you good catch